pkgs/servers/peertube/build-client.nix

```
@@ -24,20 +24,20 @@ rec {
     inherit version;
     pname = "peertube-client";
     src = sources;
+
+    nativeBuildInputs = [ nodejs esbuild ];
```

```suggestion
    nativeBuildInputs = [ nodejs ];
```
I think you should either change this to `esbuild_locked` or remove it completely. As `esbuild_locked` is explicitly referenced in `buildPhase` this should work without it.

---

Removed